http/basic/README.md

```diff
@@ -0,0 +1,209 @@
+# Generic HTTP Protocol Binding
+
+[![Javadocs](http://www.javadoc.io/badge/io.cloudevents/cloudevents-http-basic.svg?color=green)](http://www.javadoc.io/doc/io.cloudevents/cloudevents-http-basic)
+
+For Maven based projects, use the following to configure the CloudEvents Generic HTTP Transport:
+
+```xml
+<dependency>
+    <groupId>io.cloudevents</groupId>
+    <artifactId>cloudevents-http-basic</artifactId>
+    <version>2.0.0-milestone1</version>
+</dependency>
+```
+
+## Sending and Receiving CloudEvents
+To send and receive CloudEvents we use `MessageWriter` and `MessageReader`, respectively.
+This module offers generic factory methods for creation of those in `HttpMessageFactory`.
+
+```java
```

Comment:
Remove this, link to the code

Reply:
How should I reference the code? Hyperlink to github?

Reply:
yep just relative link to code, check out how to do that in other readmes